Dockerfile

```diff
@@ -1,35 +1,26 @@
-FROM ubuntu:18.04
+FROM python:3.9.5-slim
 
-ENV PYTHON_VERSION=3.9 
```

Comment:
Can we get Rod's approval for the deletions here? (Not just this one, all deletions in this file). I don't exactly understand what he had originally in this Dockerfile and would like to make sure none of this is necessary for running.

Reply:
From my understanding (please correct me if I'm wrong), using Ubuntu provides a more general Linux environment suitable for running a variety of services and applications within containers. However, since our project is specifically focused on a Python-based environment, switching to python:3.9.5-slim might be faster and more tailored to our needs. I would really appreciate Rod's opinion on this matter.

Reply:
Zhen is correct on this one.